### ISSUE TITLE
Manual Release required between GNOME and XFCE

### ISSUE THREAD

**Issue by jeremy-cxf** [feschber/lan-mouse#125]
Going between GNOME Shell 46.0 and XFCE 4.18. XFCE is running on 0.7.2 bin at the moment, where as my Ubuntu machine is compiled from source (however switching to a precompiled binary the behaviour is still the same).

I'm able to go to either desktop fine, but it requires a key-press release each time to get back to GNOME. I would like to get to a state of just being able to move to either monitor without this. Would just like to validate if this just a current limitation at the moment, or something I need to fix elsewhere.

I did find similar issues (https://github.com/feschber/lan-mouse/pull/94) but these appeared fixed in newer versions so assumed its something I'm doing or I am just misinterpreting the behavior.

Best!

**Comment by feschber** [comment]
Xfce needs X11 input capture to be able to recognize when the pointer leaves the screen, which is not yet implemented. So this is expected at the moment.

**Comment by jeremy-cxf** [comment]
Thanks for replying @feschber and for the heads up. Is it worth having a matrix or something similar in the README of current limitations between desktops? I think in my naivety coming from barrier that the behavior is somewhat ingrained into my head. Obviously hopping between different desktop APIs is going to encompass a number of challenges that might need to be spelled out to dummies like myself :')

**Comment by feschber** [comment]
Yeah I may want to update this to be a bit more cleat about what input capture and input emulation means

https://github.com/feschber/lan-mouse?tab=readme-ov-file#os-support
